pkgs/os-specific/linux/kernel/generic.nix

```diff
@@ -70,6 +70,18 @@
 assert stdenv.isLinux;
 
 let
+  # Dirty hack to make sure that `version` & `src` have
```

Comment:
OK, this doesn't seem to work with ofborg's way of detecting a position. cc @cole-h is this intended to be working? Otherwise I'd consider to just merge it now to fix the underlying issue for now :)

Reply:
If you still need the hack, do also set `pos` for other tooling and future ofborg.

Reply:
Done!

Reply:
Some ofborg behaviors rely on the old version to be set up correctly. Could that be the case with this?

Reply:
Not sure if I'm misunderstanding you, but ofborg resolves maintainers by checking which attr-paths and which files have changed and which correlate (i.e. did `<nixpkgs/pkgs/.../hello/default.nix>` and `pkgs.hello` change -> notify `pkgs.hello.meta.maintainers`, see https://github.com/NixOS/ofborg/blob/released/ofborg/src/maintainers.nix).

What I'm trying to say is, I just threw `maintainers.nix` against a local nixpkgs checkout to investigate, so I don't see how this is related to older versions as the behavior seems solely based on a single git checkout. Or am I misunderstanding you?

Reply:
So these paragraphs are not related?

```
  # Dirty hack to make sure that `version` & `src` have
  # `<nixpkgs/pkgs/os-specific/linux/kernel/linux-x.y.nix>` as position
  # when using `builtins.unsafeGetAttrPos`.
  #
  # This is to make sure that ofborg actually detects changes in the kernel derivation
  # and pings all maintainers.
```

Reply:
Indeed OfBorg doesn't read `meta.position` yet and it has an issue for it https://github.com/NixOS/ofborg/issues/297